USH_Apps/NY_CICT/Base/base.robot

```diff
@@ -0,0 +1,190 @@
+*** Settings ***
+Library  SeleniumLibrary
+Library    String
+Library    DateTime
+Resource    ../Locators/locators.robot
+Resource     ../Case Investigation (CI)/Forms/register a new case form.robot
+Resource    ../Contact Tracing (CT)/Forms/register a new contact form.robot
+Library     driverpath.py
+Library    2FA.py
+
+
+*** Variables ***
+
+# Case Search
+${first-name_case_search}    xpath:(//td/div[contains(., "First Name")]/following::input)[1]
+${last-name_case_search}    xpath:(//td/div[contains(., "Last Name")]/following::input)[1]
+${search all cases in the list}    //button[contains(., 'Search All')]
+
+*** Keywords ***
+    
+HQ Login
+    ${chromedriver_path}=   driverpath.Get Driver Path
+    ${chrome_options}=  Evaluate  sys.modules['selenium.webdriver'].ChromeOptions()  sys, selenium.webdriver
+    Call Method    ${chrome_options}    add_argument    --disable-extensions
+    # Call Method    ${chrome_options}    add_argument    --headless
```

Comment:
This line needs to be uncommented or else all the scripts will fail.

Reply:
Yes. In this framework we are referring the same. So When running on local I comment this statement and on CI uncomment the same.
@kiran-tripathy

Reply:
Uncommented!